src/Core/Content/Product/Cart/ProductLineItemCommandValidator.php

```diff
@@ -113,6 +113,10 @@ private function findProducts(array $commands)
 
         $ids = array_values(array_filter($ids));
 
+        if ($ids === []) {
```

Comment:
I usually prefer the `count` approach since the underlying data structure can change. Not sure if relevant in this case and I am also not sure if there is a performance difference.

```suggestion
        if (\count($ids) === 0) {
```